cypress/e2e/applicant/apply.cy.js

```diff
@@ -0,0 +1,226 @@
+  cy.contains("Please enter whole pounds only");
+  cy.get("[data-cy=cy-APPLICANT_AMOUNT-text-input-numeric").type("100");
+  saveAndContinue();
+
+  cy.contains("Where will this funding be spent?");
+  cy.contains(
+    "Select the location where the grant funding will be spent. You can choose more than one, if it is being spent in more than one location.",
+  );
+  cy.get('[type="checkbox"]').not("[disabled]").check().should("be.checked");
+  saveAndContinue();
+
+  cy.contains("Summary of Required checks");
+  cy.get("[data-cy=cy-section-details-APPLICANT_ORG_NAME]").contains(
+    "Enter the name of your organisation",
+  );
+  cy.contains("My First Org");
+  cy.get("[data-cy=cy-section-details-APPLICANT_TYPE]").contains(
+    "Choose your organisation type",
+  );
+  cy.contains("Limited company");
+  cy.get("[data-cy=cy-section-details-APPLICANT_ORG_ADDRESS]").contains(
+    "Enter your organisation's address",
+  );
+  cy.get("[data-cy=cy-organisation-value-APPLICANT_ORG_ADDRESS] > ul > li")
+    .should("contain", "Address line 1,")
+    .and("contain", "Address line 2,")
+    .and("contain", "Town,")
+    .and("contain", "County,")
+    .and("contain", "Postcode");
+  cy.get("[data-cy=cy-section-details-APPLICANT_ORG_CHARITY_NUMBER]").contains(
+    "Enter your Charity Commission number (if you have one) (optional)",
+  );
+  cy.get("[data-cy=cy-section-details-APPLICANT_ORG_COMPANIES_HOUSE]").contains(
+    "Enter your Companies House number (if you have one) (optional)",
+  );
+  cy.get("[data-cy=cy-section-details-APPLICANT_AMOUNT]").contains(
+    "How much does your organisation require as a grant?",
+  );
+  cy.contains("100");
+  cy.get("[data-cy=cy-section-details-BENEFITIARY_LOCATION]").contains(
+    "Where will this funding be spent?",
+  );
+  cy.get("[data-cy=cy-organisation-value-BENEFITIARY_LOCATION] > ul > li")
+    .should("contain", "North East England,")
+    .and("contain", "North West England,")
+    .and("contain", "South East England,")
+    .and("contain", "South West England,")
+    .and("contain", "Midlands,")
+    .and("contain", "Scotland,")
+    .and("contain", "Wales,")
+    .and("contain", "Northern Ireland");
+
+  yesSectionComplete();
+};
+
+describe("Find a Grant", () => {
+  beforeEach(() => {
+    // We have to visit base url first to prevent issues with cross-origin
+    cy.visit(BASE_URL);
+    // then log in to the One Login integration environment to prevent the popup appearing
+    const username = Cypress.env("oneLoginSandboxUsername");
+    const password = Cypress.env("oneLoginSandboxPassword");
+    cy.visit(
+      `https://${username}:${password}@signin.integration.account.gov.uk/sign-in-or-create`,
+      {
+        failOnStatusCode: false,
+      },
+    );
+    // then return back to the base url to execute the tests
+    cy.visit(BASE_URL);
+    cy.contains("Reject analytics cookies").click();
+  });
+
+  it("loads the page", () => {
+    cy.contains("Find a grant");
+  });
```

Comment:
...rm?

Reply:
this test takes 1s so I'd rather keep it in just in case the entire site goes down